docs/usage.md

```diff
@@ -8,6 +8,13 @@ Refer to the table of [template blocks](./docs/template-blocks.md) and their def
 
 The [govuk_template sets a skip link](https://github.com/alphagov/govuk_template/blob/master/source/views/layouts/govuk_template.html.erb#L64-L68) to `#content`, but doesn't provide an element with `id="content"`. You'll need to add `id="content"` to your main content area, to ensure the skip link will work.
 
+    <main id="content" role="main" tabindex="-1">
```

Comment:
Do we need the tag to be `main` if we've got role="main" applied? Just thinking `main` has to be shimmed in older browsers..